testProjects/mjs/index.js

```diff
@@ -0,0 +1,7 @@
+import Stripe from 'stripe';
+
+const stripe = new Stripe(process.argv[2]);
+
+stripe.customers.create({
```

Comment:
Can we import all infrastructure classes that are exposed to customers? i.e. StripeError.